### ISSUE TITLE
Missing user-user documentation on how to get survey_display_local up and running on the Asus tablet with Kiosk App

### ISSUE THREAD

**Issue by ryantanaka** [erdl/inactive-survey_display_template#4]
> Specifically, you should focus on where the app is, what it takes to run, what buttons to go full screen and so forth so get the survey_display_local going. This is user-user level. Try to imagine a scenario where the asus just crashed, and now someone who never operated the device needs to go through. I suggest you take screeenshots, and keep an eye on the compression level / size of the image so we don't waste too much space on github on it (see if < 1 MB will work resolution wise).

continuation from https://github.com/erdl/survey_display_local/issues/2

@carlosviansi  let me know when have a Wiki up for this project and I'll submit the pull request, thanks!

**Comment by ryantanaka** [comment]
The documentation is done. When you get the chance you can review it.

Link to [wiki repository clone](https://github.com/ryantanaka/survey_display_local-wiki.git)

**Comment by carlosparadis** [comment]
I am a bit confused here. Shouldn't the documentation be on this actual project wiki rather than a repository you created called wiki? 

https://github.com/erdl/survey_display_local/wiki

Are you being unable to edit?

**Comment by ryantanaka** [comment]
Yeah, I cannot edit the wiki on the actual project. I can't directly make a pull request for the project wiki either. 

If you give me privileges to edit the wiki, then I should be able to directly add those changes in. 
It's probably easier on your end too if I can edit the wiki directly right.

**Comment by ryantanaka** [comment]
Actually, sorry you can ignore that comment, I'll add the wiki to my fork of the project and then submit the pull request.

**Comment by carlosparadis** [comment]
Hey Ryan, no worries on the wiki for Pull Requests. We only need that for the code! Just add in and I will edit afterward. In fact, I prefer like so in this case since you can't submit "broken documentation", and memory of what you submit is welcomed (just don't go uploading on me credentials or large files haha).

**Comment by ryantanaka** [comment]
Ohh! okay got it. Sorry for the confusion. haha

So just to clarify: 
For submitting documentation we should use the following workflow?
-Write up documentation here and/or upload the FILE.md
-You will copy it over from here into the actual wiki of the repository

**Comment by carlosparadis** [comment]
lol no, just write directly in the wiki, thats what they are for. 

You open an issue to indicate a problem, or a new feature you're adding. The solution itself either becomes a code pull request or a wiki entry. You don't need to open an issue for every change you do on the wiki, just major sessions it is worthwhile having here what prompted you to it, and have it linked on both ends. 

Makes sense? : )

**Comment by ryantanaka** [comment]
k this makes perfect sense now, thanks for clearing that up!

Can you give me privileges to add to the wiki in this repository. I'm not able to add new pages or edit existing ones .

**Comment by carlosparadis** [comment]
Could you check for me how to do that without giving you edit permissions on the code side as well? : ) This is more of a precaution side to avoid you sending commits without being from Pull Request while trying to commit locally!

**Comment by ryantanaka** [comment]
sure, no problem I'll look into it

**Comment by ryantanaka** [comment]
I think this may be a solution. 

In the settings tab of the survey_display_local repo:
1.  options -> check "restrict editing to users in teams with push access only".
2. In "Collaborators & teams" -> give our team write access
3. In "Branches" ->  under "Protected Branches" add "master" and "dev", 
For each branch, click "edit", then check "Protect this branch" and "Require pull request reviews before merging". Check additional boxes as you feel needed. 

Aside from that, I'm not sure how to give access to the wiki without exposing team members to push/write access.

**Comment by carlosparadis** [comment]
@ryantanaka nice finding. I set it up. I want you to do two tests: 

- Try to push directly to dev a commit with a single file and see if you get denied (if you do, let me know what you get as a message). 

- Try to make a pull request for this 1 test file to dev, and then try to merge it yourself. In theory, it shouldn't allow you to. 

Ah, and of course try to see if you can do whatever you like with the wiki now!

**Comment by ryantanaka** [comment]
Okay, looks good. I tried both. Also, I was able to make a wiki and save it. I'll start moving the appropriate documentation into survey_display_server.

#### task

> Try to push directly to dev a commit with a single file and see if you get denied (if you do, let me know what you get as a message).

#### result:

> Counting objects: 3, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (2/2), done.
Writing objects: 100% (3/3), 274 bytes | 0 bytes/s, done.
Total 3 (delta 1), reused 1 (delta 0)
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
remote: error: GH006: Protected branch update failed for refs/heads/dev.
remote: error: You're not authorized to push to this branch. Visit https://help.github.com/articles/about-protected-branches/ for more information.
To https://github.com/erdl/survey_display_local.git
 ! [remote rejected] dev -> dev (protected branch hook declined)
error: failed to push some refs to 'https://github.com/erdl/survey_display_local.git'

#### task

> Try to make a pull request for this 1 test file to dev, and then try to merge it yourself. In theory, it shouldn't allow you to.

#### result:

![screenshot from 2017-09-06 14-26-26](https://user-images.githubusercontent.com/19616012/30140331-76f149e0-930f-11e7-8bc4-2032e5d53baf.png)

**Comment by carlosparadis** [comment]
Alright great! I only did this to the server and the local github, let me know if you need elsewhere!
